picachu/modules/galleries/queries/get_gallery_query.py

```diff
@@ -14,19 +14,9 @@ def by_id(cls, gallery_id: int) -> Optional[Gallery]:
         try:
             return current_session \
                 .query(Gallery) \
-                .get(gallery_id)
-        finally:
-            current_session.close()
-
-    @classmethod
-    def by_user_id(cls, current_user_id: int) -> List:
-        current_session = session()
-        try:
-            galleries_list = current_session \
-                .query(Gallery) \
-                .filter(Gallery.user_id == current_user_id) \
-                .all()
-            return galleries_list
+                .filter(Gallery.deleted_at_utc == None) \
```

Comment:
The same filtering is needed in the by_user_id method. Without it, when we get galleries, we also get deleted ones.